PhpAmqpLib/Connection/AMQPLazySSLConnection.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace PhpAmqpLib\Connection;
+
+class AMQPLazySSLConnection extends AMQPSSLConnection
+{
+     /**
+     * Gets socket from current connection
+     *
+     * @deprecated
```

Comment:
This probably will go to 3.0, so let's drop deprecated methods.

Reply:
done

Reply:
@ramunasd can we also port this change for the 2.x version? I would like to use it there as well.